spec/UserPII.spec.js

```
@@ -143,28 +132,25 @@ describe('Personally Identifiable Information', () => {
   });
 
   it('should not get PII via REST', done => {
-    request({
+    return request({
       url: 'http://localhost:8378/1/classes/_User',
       headers: {
         'X-Parse-Application-Id': 'test',
         'X-Parse-Javascript-Key': 'test',
       },
     })
-      .then(
```

an error with the request gets swallowed here.